README.md

```diff
@@ -85,6 +85,14 @@ end
 
 It can be used to validate the versions file or to [generate code](https://json-schema.org/implementations.html) from the schema.
 
+## Tools using version.json
+
+This is an (incomplete) list of tools that make use of the published `versions.json`.
+If you maintain such a tool, consider adding info about it in this list.
+This allows us to verify if changes might affect downstream tooling.
+
+- [julia-actions/setup-julia](https://github.com/julia-actions/setup-julia)
```

Comment:
```suggestion
- [julia-actions/setup-julia](https://github.com/julia-actions/setup-julia)
- [johnnychen94/jill.py](https://github.com/johnnychen94/jill.py): the Julia installer written in Python
- [abelsiqueira/jill](https://github.com/abelsiqueira/jill): the Julia installer (for Linux)
```

and also [jill.sh](https://github.com/abelsiqueira/jill/blob/442a9dc86713468f15a8df7d635e9c5b2fc4c0fd/jill.sh#L69)

cc: @abelsiqueira

Reply:
Thanks, @johnnychen94. Can you say just "Julia installer", please?

Reply:
Both `jill` and `jill.py` should probably say "a Julia installer", since there are other Julia installers as well (such as Juliaup).